packages/components/src/icon/stories/index.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Internal dependencies
+ */
+import Icon from '../';
+import { SVG, Path } from '../../';
+
+export default { title: 'Icon', component: Icon };
+
+const IconSizeLabel = ( { size } ) => <div style={ { fontSize: 12 } }>{ size }px</div>;
+
+export const _default = () => (
+	<div>
+		<Icon icon="screenoptions" />
+		<IconSizeLabel size={ 24 } />
+	</div>
+);
+
+export const Sizes = () => {
```

Comment:
I think the convention is to use `camelCase` for story names: https://storybook.js.org/docs/formats/component-story-format/.

I don't have a strong opinion, but we used that in the stories we already merged so it's just for consistency.

Reply:
@epiqueras Thank you!